client/ayon_core/hosts/houdini/plugins/create/create_usdrender.py

```diff
@@ -2,40 +2,134 @@
+        # TODO: Probably better to change the default somehow for just this
+        #    Creator on the HoudiniSubmitDeadline plug-in, if possible?
+        (
+            instance_data
+            .setdefault("publish_attributes", {})
+            .setdefault("HoudiniSubmitDeadlineUsdRender", {})["export_chunk"]
+        ) = 1000
+
         instance = super(CreateUSDRender, self).create(
             product_name,
             instance_data,
             pre_create_data)  # type: CreatedInstance
 
         instance_node = hou.node(instance.get("instance_node"))
 
-
         parms = {
             # Render frame range
             "trange": 1
         }
         if self.selected_nodes:
             parms["loppath"] = self.selected_nodes[0].path()
+
+        if pre_create_data.get("split_render", self.split_render):
+            # Do not trigger the husk render, only trigger the USD export
+            parms["runcommand"] = False
+            # By default, the render ROP writes out the render file to a
+            # temporary directory. But if we want to render the USD file on
+            # the farm we instead want it in the project available
+            # to all machines. So we ensure all USD files are written to a
+            # folder to our choice. The
+            # `__render__.usd` (default name, defined by `lopoutput` parm)
+            # in that folder will then be the file to render.
+            parms["savetodirectory_directory"] = "$HIP/render/usd/$HIPNAME/$OS"
+            parms["lopoutput"] = "__render__.usd"
+            parms["allframesatonce"] = True
+
+        # By default strip any Houdini custom data from the output file
+        # since the renderer doesn't care about it
+        parms["clearhoudinicustomdata"] = True
+
+        # Use the first selected LOP node if "Use Selection" is enabled
+        # and the user had any nodes selected
+        if self.selected_nodes:
+            for node in self.selected_nodes:
+                if node.type().category() == hou.lopNodeTypeCategory():
+                    parms["loppath"] = node.path()
+                    break
+
+        # Set default renderer if defined in settings
+        if pre_create_data.get("renderer"):
+            parms["renderer"] = pre_create_data.get("renderer")
+
         instance_node.setParms(parms)
 
-        # Lock some Avalon attributes
+        # Lock some AYON attributes
         to_lock = ["productType", "id"]
         self.lock_parameters(instance_node, to_lock)
+
+    def get_pre_create_attr_defs(self):
+
+        # Retrieve available renderers and convert default renderer to
+        # plug-in name if settings provided the display name
+        renderer_plugin_to_display_name = get_usd_rop_renderers()
+        default_renderer = self.default_renderer or None
+        if (
+                default_renderer
+                and default_renderer not in renderer_plugin_to_display_name
+        ):
```

Comment:
Is it just me or this is overindented?
```suggestion
        if (
            default_renderer
            and default_renderer not in renderer_plugin_to_display_name
        ):
```

Reply:
I agree - but it's what my IDE does based on some PEP8 and flake rules. I'm fine with either but I have a feeling a strict linter will later agree with the further indents. @iLLiCiTiT?

Reply:
The further indents are needed if `):` is on same line as conditions (don't understand why?). I agree that this is too much indented and the suggestion is correct.